tests/integration/run_tests.sh

```diff
@@ -150,12 +150,10 @@ fi
 # Set separator to newline (in case tests have spaces in their name)
 IFS=$'\n'
 
-TESTS=$("${ES_EXEC_PATH}" --tests --resources "${RESOURCES}" --config "${ES_CONFIG_TEMPLATE_PATH}")
-TESTS_OK=($(echo "${TESTS}" | grep -e "^active" | cut -f2)) || true
-TESTS_NOK=($(echo "${TESTS}" | grep -e "^known failure" -e "^missing feature" | cut -f2)) || true
```

Comment:
Just move "known failure" and "missing feature" to the OK section is all that seems needed since your previous PR.

Reply:
This comment is no longer relevant.